### DISCUSSION TITLE
Create movies example website

### DISCUSSION THREAD

**Discussion by seneca** [nuejs/nue#120] [Ideas]
It would be great to add nuejs implementation of a movie example in https://tastejs.com/movies/.
That website has a number of implementations in various frameworks. This would definitely raise awareness of nuejs project and would show off nuejs’s strengths. 

It was created by Addy Omani and I’m sure he would be more than happy to add it to that website.

**Comment by tipiirai** [comment]
Nice! I wasn't aware of this project. So much better than the crappy Todo MVC. Clearly something that needs to be implemented with Nue at some point. I'm personally focusing on templates after Nuemark / Nue CSS is finished.
